rpmlb/yaml.py

```diff
@@ -29,3 +31,18 @@ def dump(self):
         output = yaml.dump(self.content, Dumper=Dumper,
                            default_flow_style=False)
         print(output)
+
+    @staticmethod
+    def run_cmd_element(cmd_element, **kwargs):
+        cmds = None
```

Comment:
I think this line is unnecessary – you do not need to declare a variable before `if` in python:

```python
>>> if True:
...     x = 42
...
>>> x
42
```

Unfortunately, I'm not able to find exact documentation on how this works, but it does…